devtools/7/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#
+# Composer for PHP 7 Dockerfile
+#
+# https://github.com/phalcon/dockerfiles
+#
+
+# Pull base image
+FROM phalconphp/composer:7
+
+MAINTAINER Serghei Iakovlev <serghei@phalconphp.com>
+
+# Set up the application directory
+VOLUME ["/app"]
+RUN apk update && apk add git && composer global require phalcon/devtools \
```

Comment:
You can change this to :

`apk add --no-cache --update add git && composer global require phalcon/devtools`

or clear the cache in the end